client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -81,7 +83,8 @@ enum WindowActions {
 }
 
 enum EditorActions {
-	GoToAllPosts = 'goToAllPosts',
+	GoToAllPosts = 'goToAllPosts', // Unused action in favor of CloseEditor. Maintained here to support cached scripts.
```

Comment:
We're using a daily cache for JP and AT sites ([#](https://github.com/Automattic/jetpack/blob/bb9aaeb930ada902d698060f5f6e45b689df3d55/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php#L259)), so we can safely remove this action one day after landing these changes.